hi-ml/src/health_ml/utils/output_directories.py

```diff
@@ -0,0 +1,32 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from dataclasses import dataclass
+
+from param import Path
+
+
+@dataclass(frozen=True)
+class OutputFolderForTests:
+    """
+    Data class for the output directories for a given test
+    """
+    root_dir: Path
```

Comment:
replace all usage of that with the random_folder fixture that we already have in hi-ml-azure?